pkg/cache/cache_test.go

```diff
@@ -1733,6 +1733,44 @@ func TestClusterQueueUpdateCodependentResources(t *testing.T) {
 	}
 }
 
+// TestHeadsCancelled ensures that the Heads call returns when the context is closed.
+func TestHeadsCancelled(t *testing.T) {
```

Comment:
```suggestion
func TestWaitForPodsReadyCancelled(t *testing.T) {
```

Although the test looks just copied from the next one?

Reply:
That's right, now fixed.